app/models/fae/static_page.rb

```diff
@@ -42,15 +42,15 @@ def self.setup_dynamic_singleton
           languages.each do |lang|
             # Save with suffix for form fields
             define_association("#{name}_#{lang}", type)
-            define_validations("#{name}_#{lang}", type, value[:validates]) if value.try(:[], :validates).present?
```

Comment:
@jamesmk - not sure why these changes static_page diffs are showing up. I just pulled from master to bring my branch up to date with your recent changes.